plugins/BEdita/Core/src/Model/Behavior/PriorityBehavior.php

```diff
@@ -88,7 +88,12 @@ protected function maxValue(EntityInterface $entity, string $field, array $scope
     {
         $conditions = [];
         foreach ($scope as $item) {
-            $conditions[$this->getTable()->aliasField($item)] = $entity->get($item);
+            $valueField = $entity->get($item);
```

Comment:
[minor] we could use automatici `IS NULL` creation https://book.cakephp.org/3/en/orm/query-builder.html#automatic-is-null-creation 

```php
$keyField = sprintf('%s IS', $this->getTable()->aliasField($item));
$conditions[$keyField] = $entity->get($item);
```

Reply:
Nice!